Flaky Test - awsemfexporter.TestIntDataPointSliceAt

Failing in CI, example [here](https://app.circleci.com/pipelines/github/SumoLogic/opentelemetry-collector-contrib/1582/workflows/376f9b5f-6477-4f6e-bc77-2547379d66f3/jobs/13227)

```
--- FAIL: TestIntDataPointSliceAt (3.03s)
    --- FAIL: TestIntDataPointSliceAt/w/_2nd_rate_calculation (1.01s)
        datapoint_test.go:310: 
            	Error Trace:	datapoint_test.go:310
            	Error:      	Max difference between 1 and 0.9891196834817013 allowed is 0.01, but difference was 0.010880316518298683
            	Test:       	TestIntDataPointSliceAt/w/_2nd_rate_calculation
logEvents[0].Timestamp=5577006791947779410.
logEvents[1].Timestamp=8674665223082153551.
logEvents[2].Timestamp=6129484611666145821.
logEvents[3].Timestamp=4037200794235010051.
logEvents[4].Timestamp=3916589616287113937.
logEvents[5].Timestamp=6334824724549167320.
logEvents[6].Timestamp=605394647632969758.
logEvents[7].Timestamp=1443635317331776148.
logEvents[8].Timestamp=894385949183117216.
logEvents[9].Timestamp=2775422040480279449.
logEvents[0].Timestamp=605394647632969758, logEvents[1].Timestamp=894385949183117216.
logEvents[1].Timestamp=894385949183117216, logEvents[2].Timestamp=1443635317331776148.
logEvents[2].Timestamp=1443635317331776148, logEvents[3].Timestamp=2775422040480279449.
logEvents[3].Timestamp=2775422040480279449, logEvents[4].Timestamp=3916589616287113937.
logEvents[4].Timestamp=3916589616287113937, logEvents[5].Timestamp=4037200794235010051.
logEvents[5].Timestamp=4037200794235010051, logEvents[6].Timestamp=5577006791947779410.
logEvents[6].Timestamp=5577006791947779410, logEvents[7].Timestamp=6129484611666145821.
logEvents[7].Timestamp=6129484611666145821, logEvents[8].Timestamp=6334824724549167320.
logEvents[8].Timestamp=6334824724549167320, logEvents[9].Timestamp=8674665223082153551.
FAIL
FAIL	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awsemfexporter	16.711s
```

--- a/exporter/awsemfexporter/datapoint_test.go
+++ b/exporter/awsemfexporter/datapoint_test.go
@@ -307,7 +307,7 @@ func TestIntDataPointSliceAt(t *testing.T) {
 			assert.Equal(t, 1, dps.Len())
 			dp := dps.At(0)
 			if strings.Contains(tc.testName, "2nd rate") {
-				assert.InDelta(t, expectedDP.Value.(float64), dp.Value.(float64), 0.01)
+				assert.InDelta(t, expectedDP.Value.(float64), dp.Value.(float64), 0.02)
 			} else {
 				assert.Equal(t, expectedDP, dp)
 			}

